public/client.js

```diff
@@ -8,6 +8,7 @@ var input = document.getElementById("input");
 // get list of games
 
 socket.emit("getListOfGames");
+socket.emit("playerStartsGame", 1);
```

Comment:
This is happening when the client opens the JS file, it should happen on a button click. For an example of how to do this check out https://github.com/merryt/acquire/blob/main/public/client.js#L30-L32